samples/dymaptic.GeoBlazor.Core.Sample.Wasm/dymaptic.GeoBlazor.Core.Sample.Wasm.csproj

```diff
@@ -3,19 +3,20 @@
     <PropertyGroup>
         <TargetFramework>net7.0</TargetFramework>
         <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+        <UserSecretsId>0d722028-1f33-4f5f-829c-e1333fd73878</UserSecretsId>
```

Comment:
user secrets do not work AFAIK in wasm, hence why these were not here before.

Reply:
actually, they do work in both server and wasm.

Reply:
both references are now removed from csproj files